cluster/manager_delete.go

```diff
@@ -30,6 +32,38 @@ func (m *Manager) DeleteCluster(ctx context.Context, cluster CommonCluster, forc
 	return nil
 }
 
+func deleteAllResource(kubeConfig []byte, logger *logrus.Entry) error {
+	client, err := helm.GetK8sConnection(kubeConfig)
+	if err != nil {
+		return err
+	}
+
+	type resourceDeleter interface {
+		Delete(name string, options *metav1.DeleteOptions) error
+	}
+
+	services := []resourceDeleter{
+		client.CoreV1().Services(""),
+		client.AppsV1().Deployments(""),
+		client.AppsV1().DaemonSets(""),
+		client.AppsV1().StatefulSets(""),
+		client.AppsV1().ReplicaSets(""),
+	}
+
+	options := metav1.NewDeleteOptions(0)
+
+	for _, service := range services {
+		for i := 0; i < 3; i++ {
+			err := service.Delete("", options)
```

Comment:
same here

Reply:
This is not a namespace it is the resource name

Reply:
Sorry you are right:)